sentinel-core/src/main/java/com/alibaba/csp/sentinel/Constants.java

```diff
@@ -70,5 +70,19 @@
      */
     public static volatile boolean ON = true;
 
+    /**
+     * Order of default processor slots
+     */
+    public static final int ORDER_NODE_SELECTOR_SLOT = -10000;
+    public static final int ORDER_CLUSTER_BUILDER_SLOT = -9000;
+    public static final int ORDER_LOG_SLOT = -8000;
+    public static final int ORDER_STATISTIC_SLOT = -7000;
+    public static final int ORDER_AUTHORITY_SLOT = -6000;
+    public static final int ORDER_SYSTEM_SLOT = -5000;
+    public static final int ORDER_GATEWAY_FLOW_SLOT = -4000;
```

Comment:
It's not a good practice setting unrelated constant here. You'd better put it in gateway adapter package. If you feel a little unsure to decide the value of it in separated package you can take it as a `user-defined` implementation out of sentinel-core.

Reply:
The two constant `ORDER_GATEWAY_FLOW_SLOT`,`ORDER_PARAM_FLOW_SLOT` have been removed here.